src/Symfony/Component/HttpKernel/Exception/DependencyMismatchException.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Exception;
+
+use RuntimeException;
```

Comment:
Hi, you mean removing DependencyMismatchException?
That was the original case before this review thread:
https://github.com/symfony/symfony/pull/13945#discussion-diff-26562986

or did you imply inline `extends \RuntimeException`